### PR TITLE
refactor: improve auctionResultsConnection interface [DO NOT MERGE]

### PR DESCRIPTION
**Pull request by MounirDhahri**
This is going to be a non-breaking change to our frontend since we still did not roll this change out to App.

What's needed before merging this
1. ✅ Revert PR on Eigen: https://github.com/artsy/eigen/pull/7867
2. ✅ Revert PR on Force: https://github.com/artsy/force/pull/11575#pullrequestreview-1220973769

### CODE REVIEW

File: src/schema/v2/me/myCollectionAuctionResults.ts [hunk 1]
```diff
@@ -12,49 +12,49 @@ import { pageable } from "relay-cursor-paging"
...
-      description: "Filter auction results by organizations",
-    },
-    sizes: {
-      type: new GraphQLList(ArtworkSizes),
-      description: "Filter auction results by Artwork sizes",
+    allowEmptyCreatedDates: {
+      type: GraphQLBoolean,
+      defaultValue: true,
+      description:
+        "Filter auction results by empty artwork created date values",
```
Comment (olerichter00) [inline src/schema/v2/me/myCollectionAuctionResults.ts:30]: **suggestion:** Not quite sure if the description is matching the param name and describing what the param does. (allow vs. filtering).

**Review by olerichter00 — APPROVED** [review]
Thanks for the adjustment 🙏

File: src/schema/v2/me/myCollectionAuctionResults.ts (hunk 1 above)
Comment (MounirDhahri) [reply src/schema/v2/me/myCollectionAuctionResults.ts:30]: this was there for a while though, I only updated the order of the args

**Review by olerichter00 — APPROVED** [review]
Nice 🌟

**Comment by MounirDhahri** [comment]
This PR has breaking changes - this shouldn't break any existing clients since we already removed the fields usage from Force and we haven't yet released the changes in App

**Pull request merged**